change-log.txt

```diff
@@ -1,4 +1,4 @@
-v. 8.3.13 11 Mar 2020 keep daily log of uploads per blog site and include it in the email to the user
+v. 8.3.13 16 Mar 2020 fixed bug to prevent submission of webonary admin plugin page when an Enter is input on a text field 

```

Comment:
It looks like you removed the old 8.3.13. Was that intentional?

Reply:
It was, but perhaps it should not have been? Are we using each entry in the change log to indicate a merge into master, or a new release into production? 

Also, since the product owner requested that we not send the entire daily log, I backed out of that code, but still will keep the log for future usage. 

If you think it would be helpful to keep a line in the change log for each merge to master, I can put it back in.

Thanks!

Reply:
I was just making sure it wasn't accidental.